schemaregistry/internal/rest_service.go

```diff
@@ -36,6 +36,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/confluentinc/confluent-kafka-go/v2/kafka"
```

Comment:
Can we avoid the dependency of the SR client on Kafka?

Reply:
Go client version is currently tied to librdkafka version, so we need to import Kafka to use `C.rd_kafka_version()`   to get the version from librdkafka C binding. We can either:
1. Use a hardcoded constant to record client version in SR module, and add notes to manually bump it
2. Keep it as it is, as SR client already has dependency on Kafka: https://github.com/confluentinc/confluent-kafka-go/blob/bb44f4b44ab8bc1d772b6167c432800cea7a9eda/schemaregistry/serde/avro/avro_generic.go#L21

Personally I prefer 2. Manual bumping is hard to maintain and it's easily overlooked